### DISCUSSION TITLE
据最近空洞异闻，虚数之树说不定真的存在。

### DISCUSSION THREAD

**Discussion by demolute** [share121/inter-knot#2791] [其他]
相信有不少人听说有人在空洞里看见了一个和自己长得相似的人。有专家推测，这可能是空洞在五维空间引发的紊乱，导致人们看见了其他世界线的自己。据专家大胆猜测，人们可以利用空洞对虚数之树进行有效观测

**Comment by OttoApocalypse01** [comment]
不要观测！不要观测！不要观测！<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Reply by demolute** [reply]
这对于科学界来讲意义重大，我们不应该于此止步不前

**Reply by AnonCN** [reply]
回头！这不是未来！

**Comment by lixiao189** [comment]
有没有空洞的背后是虚无的灾害，但是在神明视线的死角下，仍有人想要做梦，我们绳匠绝非滤尽神选者后的残渣！v 我 50，封你做深渊教团司铎
